### PR TITLE
VIH-7389 Participant will not connect to call unless connected to event hub

### PR DESCRIPTION
**Pull request by FarronF**
### JIRA link (if applicable) ###
VIH-7389

### Change description ###
User will only connect to video call if eventhub connection is active. If inactive and reconnects will connect to last call.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts [hunk 1]
```diff
@@ -784,9 +794,16 @@ export abstract class WaitingRoomBaseDirective {
             conference: this.conferenceId,
             participant: this.participant.id
         };
-        this.logger.debug(`${this.loggerPrefix} Conference has setup`, logPayload);
-        this.videoCallService.connect('', null);
-        this.outgoingStream = callSetup.stream;
+        console.log(this.eventService);
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts:797]: might want to remove this console logging :)

**Comment by FarronF** [comment]
This doesn't work correctly for participants in a vmr will need to be redone.

**Pull request closed without merging**